script/setup/install-runc

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+#
+# Builds and installs runc to /usr/local/go/bin based off
+# the commit defined in RUNC.md
+#
+set -eu -o pipefail
+
+RUNC_COMMIT=$(grep opencontainers/runc vendor.conf | cut -d " " -f 2)
```

Comment:
@AkihiroSuda what do you think about this?

Reply:
Generally vendor.conf != RUNC.md, but I think it is ok to call it a day